src/index.js

```diff
@@ -1,22 +1,69 @@
 import '../node_modules/bootstrap/dist/js/bootstrap.bundle.js';
 import './scss/style.scss';
 import firstDisplay from './jsmodules/array.js';
```

Comment:
- [REMINDER] Remember to remove this importation and refactor any part of code that uses it.

Reply:
Thanks for the reminder, I removed them all.